packages/component-library/package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@kaizen/deprecated-component-library-helpers": "^2.5.0",
-    "@kaizen/draft-button": "^5.6.2",
+    "@kaizen/button": "^1.0.1",
```

Comment:
I think we might need to dig into this one. The usages of `Button` don't appear to be actual usages, but there's certainly button-ish references within the files (eg. the readme)